_guide/tools.md

```diff
@@ -108,9 +108,9 @@ Evaluate WAI-ARIA according to the [ARIA in HTML](https://www.w3.org/TR/html-ari
 *   [Contrast Finder by Tanaguru](https://contrast-finder.tanaguru.com/) - [Contrast Finder GitHub Page](https://github.com/Tanaguru/Contrast-Finder)
 *   [Contrast-Ratio.com](https://contrast-Ratio.com) - [Contrast-Ratio GitHub Page](https://github.com/siege-media/contrast-ratio) \
 Allows for testing of contrast of text on top of a semi transparent background.
-*   [Contrast Checker](https://contrast-checker.glitch.me/) - [Contrast Checker on Glitch](https://glitch.com/edit/#!/contrast-checker?path=server.js%3A1%3A0)
```

Comment:
@mgifford I think we should drop the Glitch link as it is not loading for me (it is glitching pun intended). Use just https://contrast-checker.netlify.app/.

Reply:
Fixed this one